src/couch_mrview/src/couch_mrview_test_util.erl

```diff
@@ -13,6 +13,7 @@
 -module(couch_mrview_test_util).
 
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
In this case `-compile(export_all).` and `-compile(nowarn_export_all).` is appropriate. Since all implemented functions are used externally in the test suite.